src/chexus/validators.py

```diff
@@ -243,14 +243,16 @@ def applies_to(self, node: Dataset | Group) -> bool:
     def validate(self, node: Dataset | Group) -> Violation | None:
         import scipp as sc
 
-        if 'units' in node.children.get('rotation_speed').attrs:
-            unit = node.children.get('rotation_speed').attrs.get('units')
-            try:
-                sc.scalar(1, unit=unit).to(unit='Hz')
-            except sc.UnitError:
-                pass
-            else:
-                return
+        if 'NXlog' == node.children.get('rotation_speed').attrs.get('NX_class'):
```

Comment:
What if `rotation_speed` is not a `Group`? Could it be a simple `Dataset` if not time dependent?

Reply:
Yes it could be, but not at ess. Do you prefer it to be able to handle both cases?

Reply:
Yes.